scielomanager/thrift/scielomanager.thrift

```diff
@@ -91,6 +95,146 @@ struct Article {
 }
```

Comment:
@jamilatta, há o atributo **timestamp** que serve justamente para armazenar a data de modificação do registro. Veja aqui https://github.com/scieloorg/scielo-manager/blob/beta/scielomanager/scielomanager/connectors/article_mapping.json

-Gustavo

On 8 de junho de 2016 at 11:27:18, Jamil Atta Junior (notifications@github.com) wrote:
> > @@ -91,6 +95,146 @@ struct Article {
> > }
>  
> @fabiobatalha
>  
> Sugiro termos as datas de **criação** e **atualização** para cada entidade com o objetivo  
> podermos realizar coletas parciais.
>  
> No caso do site isso seria de suma importância.
>  
>  
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/scieloorg/scielo-manager/pull/1288/files/10e8cfcea6490348458bc8b6cccc679656e40e38#r66265725